src/overlays/actors/ovl_En_Rd/z_en_rd.c

```diff
@@ -534,25 +521,21 @@ void EnRd_Grab(EnRd* this, GlobalContext* globalCtx) {
     }
 
     switch (this->grabState) {
-        case EN_RD_GRAB_INITIAL_DAMAGE:
+        case REDEAD_GRAB_INITIAL_DAMAGE:
             Animation_PlayLoop(&this->skelAnime, &gGibdoRedeadGrabAttackAnim);
             this->grabState++;
             globalCtx->damagePlayer(globalCtx, -8);
             func_800AA000(this->actor.xzDistToPlayer, 0xFF, 1, 0xC);
             this->grabDamageTimer = 20;
-            // fallthrough
-
-        case EN_RD_GRAB_START:
+        case REDEAD_GRAB_START:
```

Comment:
I think `// fallthrough` comments can/should stay, I think @fig02 only refered to the empty line after it being unneeded but I may be wrong

Reply:
right, yeah sorry if that wasnt clear. Comments definitely help to make the fall through clear. I just like keeping the cases bunched together when they fall through into each other without a line break